draft-pda-protocol.md

```diff
@@ -703,6 +730,7 @@ unique client report. Sub-requests are structured as follows:
 ~~~
 struct {
   Time time; // Equal to PDAReport.time.
+  Extension extensions<8..2^16-1>;
```

Comment:
Any reason this is `<8..2^16-1>` here but `<4..2^16-1>` in `PDAUploadReq`?

Reply:
Me not making the change in two places :)

Reply:
```suggestion
  Extension extensions<4..2^16-1>;
```